bin/triton-mysql.py

```diff
@@ -205,13 +205,13 @@ def put_backup(self, backup_id, infile):
 
 class Containerpilot(object):
     """
-    Containerpilot config is where we rewrite Containerpilot's own config
+    Containerpilot config is where we rewrite ContainerPilot's own config
```

Comment:
We can always just capitalize the `Containerpilot` class too... it's perfectly idiomatic Python to do so.